sonar-text-dotnet/src/SonarLint.Secrets.DotNet/SonarLint.Secrets.DotNet.csproj

```diff
@@ -0,0 +1,62 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net472</TargetFramework>
```

Comment:
FYI SonarLint currently requires this to be `net472`, as that is what is used by the oldest version of VS supported by SLVS (currently VS2019 Update 3).